### ISSUE TITLE
Provide example swarm.yml with documentation on how to install

### ISSUE THREAD

**Issue by smhc** [jenkinsci/docker-swarm-plugin#2]
DockerSwarmCloud.java, initFromYaml, allows loading in a file 'jenkins.root/pluginConfigs/swarm.yml'.
However this is difficult to construct without a template.

It would be great to have an example provided.

**Comment by DanielBCoffaro** [comment]
Yes, please. I have been trying to create the swarm.yml file and figure out how to get it loaded. Information on this would be very helpful.
